bert-text-classifier/examples/test.html

```diff
@@ -0,0 +1,14 @@
+<meta charset="utf-8"/>
+<html>
+    <head>
+        <title>text classifier</title>
+        <script src="/Users/tedchang/pub_git/max-tfjs-models/bert-text-classifier/dist/src/max.sentimentclass.js"></script>
+        <script>
+        textSentimentClassifier.encode("i like strawberries").then(ids => console.log(ids));
+        textSentimentClassifier.predict("i like strawberries").then(res => console.log(res));
```

Comment:
took over 4min (not sure if my network or just the number of files to download) but maybe the UI can show some indication of loading/running so user knows something is happening